src/types/index.ts

```diff
@@ -1,46 +1,408 @@
-// @ts-ignore
-interface AutoComplete {}
+// -----------------------------------------------------------------------------
+//  --------------------------------- TYPES ------------------------------------
+// -----------------------------------------------------------------------------
 
-interface StopGroup {}
+type GroundPlacesObject = Record<Gpuid, StopGroup | StopCluster>;
 
-interface StopCluster {}
+type Gpuid = string;
 
-interface StopGroupProperties {}
+type StopGroupGpuid = Gpuid;
 
-interface StopClusterProperties {}
+type StopClusterGpuid = Gpuid;
 
-interface Filters {}
+// -----------------------------------------------------------------------------
+//  ------------------------------- INTERFACES ---------------------------------
+// -----------------------------------------------------------------------------
 
-interface GroundPlacesDiff {}
+interface StopGroup {
+  country_code: CountryCode;
+  name: string;
+  longitude: number;
+  latitude: number;
+  type: GroundPlaceType;
```

Comment:
a stopgroup can be of type stopgroup and stopcluster ?

Reply:
Nope, this can be only of type group in this case, fixed thanks!